config/manager/kustomization.yaml

```diff
@@ -12,5 +12,5 @@ apiVersion: kustomize.config.k8s.io/v1beta1
 kind: Kustomization
 images:
 - name: controller
-  newName: kueue
-  newTag: latest
+  newName: registry.cn-hangzhou.aliyuncs.com/kubernetes-image-hub/kube-queue
```

Comment:
is this intentional? lets keep it as is.

Reply:
Sorry. It's a mistake. Fixed.

Reply:
It's auto-edit ...... Sorry